x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/domain_added_to_google_workspace_trusted_domains.json

```diff
@@ -33,5 +33,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 8
+  "version": 10
```

Comment:
As with the above, looks to be an unnecessary version bump. Looking at the previous `8.2` PR _only the version` was changed there as well: https://github.com/elastic/kibana/pull/129526/files#diff-efa9b77b6dea72f367c586de660df1603e8f913d5d91ca633553f3f9a992504bR36

Note that it was a double version hope from `6` to `8` then as well, so definitely sounds like we've got a bug somewhere in the Kibana PR script?

Reply:
Double bump is the result of manual bumps done from the following.

https://github.com/elastic/detection-rules/pull/1870/files

Reply:
Double bump is the result of manual bumps done from the following.

https://github.com/elastic/detection-rules/pull/1870/files